airflow/ui/src/components/FlexibleForm/index.tsx

```diff
@@ -22,5 +22,6 @@ export type FlexibleFormElementProps = {
 };
 
 export const flexibleFormDefaultSection = "Run Parameters";
+export const flexibleFormExtraFieldSection = "Extra Field";
```

Comment:
Oh, just another small nit :-D - would recommend plural here.
```suggestion
export const flexibleFormExtraFieldSection = "Extra Fields";
```